pages/admin/dashboard/toolBuilder.tsx

```diff
@@ -170,6 +207,203 @@ const ToolBuilder: Page = () => {
         getAllTools();
     }, []);
 
+    const getMinSum = (questions) =>
```

Comment:
Try to move these functions outside of the component itself you don't need to use any closure values

Reply:
Any functions that need to interact with the component's state can stay inside the component body